src/components/ImageRows.tsx

```diff
@@ -1,62 +1,48 @@
 import { FC } from "react"
+import { Divider, VStack, Box, SimpleGrid } from "@chakra-ui/react"
 import { LabelMd } from "./Typography"
 import { Image, ImageProps } from "./Image"
-import { Divider, Flex, Show, Stack } from "@chakra-ui/react"
 
-interface ImageRowData {
+interface ImageRowItem {
+  name: string
+  image: ImageProps
+}
+
+interface ImageRowData<T extends ImageRowItem> {
   title: string
-  images: {
-    name: string
-    image: ImageProps
-  }[]
+  images: T[]
 }
 
-interface ImageRowsProps {
-  data: ImageRowData[]
-  renderImageCallback?: (imageRowData: ImageRowData) => {}
+interface ImageRowsProps<T extends ImageRowItem> {
+  data: ImageRowData<T>[]
+  renderImageCallback?: (imageRowData: T) => {}
 }
 
-export const ImageRows: FC<ImageRowsProps> = ({ data }) => {
+const DefaultItem: FC<ImageRowItem> = ({ name, image }) => (
+  <Image {...image} alt={name} />
+)
+
+const renderDefaultItem = (item: ImageRowItem) => (
+  <DefaultItem key={item.name} {...item} />
+)
+
+export const ImageRows = <T extends ImageRowItem>({
+  data,
+  renderImageCallback,
+}: ImageRowsProps<T>) => {
   return (
-    <>
-      {data.map((item, index, array) => {
+    <VStack spacing="14">
+      {data.map((item) => {
         return (
-          <>
-            <Stack direction={{ base: "column", xl: "row" }}>
-              <Stack
-                direction={{ base: "column", xl: "row" }}
-                alignItems={{ base: "flex-start", xl: "center" }}
-              >
-                <LabelMd
-                  width={{ base: "500px", xl: "150px" }}
-                  color={{ base: "brand.300", xl: "gray.300" }}
-                >
-                  {item.title}
-                </LabelMd>
-                <Show below={"xl"}>
-                  <Divider
-                    orientation="horizontal"
-                    borderTop="1px solid"
-                    color={"brand.700"}
-                  />
-                </Show>
-              </Stack>
-              <Flex wrap={"wrap"}>
-                {item.images.map((imageData) => {
-                  return (
-                    <Image
-                      {...imageData.image}
-                      marginInlineStart="0"
-                      px={"2px"}
-                    />
-                  )
-                })}
-              </Flex>
-            </Stack>
-            {index !== array.length - 1 && <Divider color="gray.300" />}
-          </>
+          <Box key={item.title}>
+            <LabelMd color="brand.300">{item.title}</LabelMd>
+            <Divider borderColor="brand.700" mb="10" />
+            <SimpleGrid columns={4} spacing="14">
+              {item.images.map(renderImageCallback ?? renderDefaultItem)}
+            </SimpleGrid>
+          </Box>
```

Comment:
Non-blocking:

I believe that having the icons placed to the left side looks better since it's aligned with the title better on smaller screens.

Before:
<img width="538" alt="image" src="https://github.com/threshold-network/splash-page/assets/40306834/1cbeb462-752d-4507-a8e7-0324d9f6c838">
_Notice how Minters title is perfectly aligned with the main title_

Now:
<img width="1114" alt="image" src="https://github.com/threshold-network/splash-page/assets/40306834/394ad75f-a0bd-4290-9e23-c2f1f8e78b71">
_The title is placed on the left and the logos are in the center which for me looks a little bit strange_

I'm fine with either solution so I'm marking it as a non-blocking. Let me know what you think.

Reply:
3189427